src/ModelContextProtocol/Protocol/Transport/SseClientTransportOptions.cs

```diff
@@ -5,6 +5,36 @@
 /// </summary>
 public record SseClientTransportOptions
 {
+    /// <summary>
+    /// The base address of the server for SSE connections.
+    /// </summary>
+    public required Uri Endpoint
+    {
+        get;
+        init
+        {
+            if (value is null)
+            {
+                throw new ArgumentNullException(nameof(value), "Endpoint cannot be null.");
+            }
+            if (!value.IsAbsoluteUri)
+            {
+                throw new ArgumentException("Endpoint must be an absolute URI.", nameof(value));
+            }
+            if (value.Scheme != Uri.UriSchemeHttp && value.Scheme != Uri.UriSchemeHttps)
+            {
+                throw new ArgumentException("Endpoint must use HTTP or HTTPS scheme.", nameof(value));
+            }
+
+            field = value;
+        }
+    }
+
+    /// <summary>
+    /// Description of the server process used for logging purposes.
```

Comment:
On "SseClientTransportOptions", "Description" is a description of the server process?